### PR TITLE
Configure PG connection from environment values.

### PR DESCRIPTION
**Pull request by rnubel**
:construction: 

When testing out `go-bootstrap`, I ran into a few issues getting the postgres connection to work. Since all of the values are hard-coded to defaults, it might be useful to allow them to be overridden by environment variables.

This change does that, while still defaulting if the environment variables are missing. However, I'm not sure if a configuration file (ala Rails' `config/database.yml`) would be a better idea, or if there's a cleaner way to make these changes (I'm still fairly new to Go).

### CODE REVIEW

**Comment by rnubel** [comment]
Also, `PGHOST` doesn't work as expected when it's set to a Unix domain socket (to my surprise, on my machine, it was set to `/tmp`), so perhaps using a different variable would be safer.

**Comment by relistan** [comment]
This would fix part of #3

File: main.go [hunk 1]
```diff
@@ -98,12 +98,29 @@ func main() {
...
+		pghost = "localhost"
+	}
+
+	if pgport == "" {
+		pgport = "5432"
+	}
+
 	for _, name := range []string{repoName, repoName + "-test"} {
-		pgDSN := fmt.Sprintf("postgres://%v@localhost:5432/%v?sslmode=disable", u.Username, name)
+		pgDSN := fmt.Sprintf("postgres://%v@%v:%v/%v?sslmode=disable", pguser, pghost, pgport, name)
```
Comment (relistan) [inline main.go]: I'd suggest flagging `sslmode` as an envvar with default "disable" as well.

**Comment by rnubel** [comment]
@relistan: I originally thought the same thing, but actually, this wouldn't fix any of #3 -- it's just the bootstrap script, which you wouldn't be running from Docker. We need to make the same set of environment changes in the *actual* app, around [here](https://github.com/go-bootstrap/go-bootstrap/blob/ced05fc3d19ed6f601dc9314f799d5aacd6eccc6/blank/main.go#L33). Which seems a bit redundant, and prompted me to write up #13.

**Comment by didip** [comment]
@rnubel Yeah, the project focuses on the creation of a new project. But the default DSN for first time migration can be made more robust.

Do you think this commit: https://github.com/go-bootstrap/go-bootstrap/commit/6fb562e227e0092f1f59237079989b0f41019ba4 resolves this issue?

I also make sure the host port separator to work with unix domain socket.

**Comment by didip** [comment]
I've applied the change in different commit.

**Pull request closed without merging**